x-pack/plugins/canvas/public/state/reducers/workpad.js

```diff
@@ -32,6 +40,10 @@ export const workpadReducer = handleActions(
     [setWriteable]: (workpadState, { payload }) => {
       return { ...workpadState, isWriteable: Boolean(payload) };
     },
+
+    [setWorkpadStyle]: (workpadState, { payload }) => {
+      return { ...workpadState, style: payload };
```

Comment:
This is sort of a nit, but can you change style -> css. I ask because our page object has a `style` property that is an object, but this is a string.